pkg/hubble/relay/observer/server.go

```diff
@@ -173,10 +175,11 @@ func (s *Server) GetNodes(ctx context.Context, req *observerpb.GetNodesRequest)
 			status, err := client.ServerStatus(ctx, &observerpb.ServerStatusRequest{})
 			if err != nil {
 				n.State = relaypb.NodeState_NODE_ERROR
-				s.opts.log.WithFields(logrus.Fields{
-					"error": err,
-					"peer":  p,
-				}).Warning("Failed to retrieve server status")
+				s.opts.log.Warn(
+					"Failed to retrieve server status",
+					logfields.Error, err,
+					logfields.Peer, p,
```

Comment:
unrelated to the patch, but shouldn't this be
```suggestion
					logfields.Peer, p.Name,
```
for consistency?

Reply:
I "blindly" converted to slog without trying to modify too much from the original code. Since you are a hubble reviewer I also agree. Thus, I'll write it with `p.Name`

Reply:
Thank you!